install/first_google_boot.sh

```diff
@@ -233,6 +235,35 @@ function extract_spinnaker_kube_credentials() {
   fi
 }
 
+function extract_spinnaker_gcr_credentials() {
+  local config_path="$GCR_DIR/gcr.json"
+  mkdir -p $(dirname $config_path)
+  chown -R spinnaker:spinnaker $(dirname $config_path)
+
+  if clear_metadata_to_file "gcr_json" $config_path; then
+    # This is a workaround for difficulties using the Google Deployment Manager
+    # to express no value. We'll use the value "None". But we don't want
+    # to officially support this, so we'll just strip it out of this first
+    # time boot if we happen to see it, and assume the Google Deployment Manager
+    # got in the way.
+    sed -i s/^None$//g $config_path
+    if [[ -s $config_path ]]; then
+      chmod 600 $config_path
```

Comment:
Sounds good, I grabbed this from the kubeconfig setup above and will change it there too.